config/default.yml

```diff
@@ -2343,6 +2343,8 @@ Lint/TopLevelReturnWithArgument:
   Description: 'Detects top level return statements with argument.'
   Enabled: true
   VersionAdded: '0.89'
+  Exclude:
+    - '**/*.jb'
```

Comment:
It would be useful to leave a comment with the reason for excluding it by default.